Editor/Editors/CinemachineVirtualCameraEditor.cs

```diff
@@ -136,6 +136,14 @@ public override void OnInspectorGUI()
             DrawTargetsInInspector(FindProperty(x => x.m_Follow), FindProperty(x => x.m_LookAt));
             DrawPropertyInInspector(FindProperty(x => x.m_StandbyUpdate));
             DrawLensSettingsInInspector(FindProperty(x => x.m_Lens));
+            DrawPropertyInInspector(FindProperty(x => x.m_Transitions));
+#if CINEMACHINE_UNITY_INPUTSYSTEM
+            var vcam = Target;
+            if (vcam.ParentCamera as CinemachineFreeLook == null && vcam.RequiresUserInput())
```

Comment:
It's needed, because Freelook has 3 vcams as children. These will each have a button without the check.
![Freelook](https://user-images.githubusercontent.com/57672095/112564574-e83d6e80-8db1-11eb-8446-87a0a31c4597.png)

Reply:
Yes I see, but you still shouldn't be checking here for specific parent types.  That's a hacky approach that breaks abstractions.  There is already a mechanism in place to suppress portions of the vcam inspector (e.g. there is no "Solo" button or "Standby Update" displayed here, etc).  You should find out how that works use the same mechanism.

Reply:
Perhaps even this note/button could be part of the header (see DrawHeaderInInspector())